### ISSUE TITLE
Fix `function-no-unknown` false positive for `palette-mix`

### ISSUE THREAD

**Issue by Mouvedia** [stylelint/stylelint#7623]
### What minimal example or steps are needed to reproduce the bug?

```css
.bar { foo: palette-mix(); }
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "function-no-unknown": true
  }
}
```

### How did you run Stylelint?

https://stylelint.io/demo

### What actually happened?

`Unexpected unknown function "palette-mix"`

<hr/>

https://caniuse.com/mdn-css_properties_font-palette_palette-mix_function
niksy/css-functions-list/issues/7

**Comment by ybiquitous** [comment]
@Mouvedia Thanks for the report. Sounds good 👍🏼 

Ref: https://developer.mozilla.org/en-US/docs/Web/CSS/font-palette/palette-mix

**Comment by Mouvedia** [comment]
🎉 
We have the green light of @niksy to fix it upstream; no more exceptions on our side.

**Comment by Mouvedia** [comment]
@ybiquitous can you trigger a dependabot scan for https://www.npmjs.com/package/css-functions-list/v/3.2.2?

**Comment by ybiquitous** [comment]
@Mouvedia Opened #7640
